src/cascadia/TerminalApp/App.cpp

```diff
@@ -875,6 +877,19 @@ namespace winrt::TerminalApp::implementation
         control.CopySelectionToClipboard(trimTrailingWhitespace);
     }
 
+    // Method Description:
+    // - Paste text from the Windows Clipboard to the focused terminal
+    // Arguments:
+    // - <none>

```

Comment:
nit: please remove line 882/883 since we don't really need them. :)

Reply:
Fixed in commit

Reply:
@DHowett-MSFT @zadjii-msft I thought you would enjoy this! 😄 #890 (note - discussion around function documentation and specifically Arguments and return values when to include them / not to include them)   Currently lots of good opinions.  🤕